chapter_multilayer-perceptrons/dropout_vn.md

```diff
@@ -124,14 +136,18 @@ The authors argue that neural network overfitting is characterized by a state in
 Dropout, they claim, breaks up co-adaptation just as sexual reproduction is argued to break up co-adapted genes.
 -->
 
-*dịch đoạn phía trên*
+Để nói cho rõ, mối liên kết đến Bishop là của chúng tôi tự đặt ra.
+Đáng ngạc nhiên, bài báo gốc về dropout xây dựng trực giác bằng việc so sánh nó với sinh sản hữu tính.
+Các tác giả cho rằng hiện tượng quá khớp mạng nơ-ron là biểu hiện của việc mỗi tầng đều dựa vào một khuôn mẫu nhất định của các giá trị kích hoạt ở tầng trước đó, và họ gọi trạng thái này là `đồng thích nghi`.
+Họ khẳng định rằng dropout phá bỏ sự đồng thích nghi này, tương tự như việc lập luận rằng sinh sản hữu tính phá bỏ các gen đã đồng thích nghi.
 
 <!--
 The key challenge then is *how* to inject this noise.
 One idea is too inject the noise in an *unbiased* manner so that the expected value of each layer---fixing the others equal to the value it would have taken absent noise.
 -->
 
-*dịch đoạn phía trên*
+Thách thức chính bây giờ là làm thế nào để thêm nhiễu.
+Một ý tưởng là: thêm nhiễu một cách *không thiên lệch* để giá trị kỳ vọng của mỗi tầng <!-- hic câu này chả hiểu gì mọi người giúp với ạ -->
```

Comment:
```suggestion
Ta có thể tiêm nhiễu vào theo một cách *không thiên lệch* miễn sao giá trị kì vọng tại mỗi tầng vẫn sẽ bằng giá trị kì vọng của chính tầng đó trước khi được tiêm nhiễu, giả sử rằng các tầng khác đang được giữ nguyên.  
```
Mình hiểu thế này. ông tác giả thiếu 3 dấu - chỗ `so that the expected value of each layer ---fixing the others--- equal to the value it would have taken absent noise.`

Reply:
à hiểu rồi, hôm đấy do mình không đọc kĩ đoạn dưới :)))
mình nghĩ vẫn nên dịch là `thêm nhiễu` để nghe đỡ lạ
```suggestion
Ta có thể thêm nhiễu một cách *không thiên lệch* sao cho giá trị kỳ vọng của mỗi tầng bằng giá trị kỳ vọng của chính tầng đó trước khi được thêm nhiễu, giả sử rằng các tầng khác được giữ nguyên.  
```